recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+c_compiler:
+  - clang_bootstrap  # [osx]
```

Comment:
necessary since `clang>=15` and `libllvm14>=15` depends on `libxml2` which requires `libiconv` and introduces a circular dependency.